pyproject.toml

```diff
@@ -0,0 +1,105 @@
+[build-system]
+requires = ["setuptools", "setuptools_scm[toml]"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "edk2-pytool-extensions"
+authors = [
```

Comment:
I think this should be a generic account (like `edk2_pytools@microsoft.com`). We have a lot of stale personal contact info in various places today.

Reply:
I have updated the email and made the request. I will hold off on merging this until the account is created.